src/Classes/ViewHelpers/DinoViewHelper.php

```diff
@@ -38,11 +38,40 @@ public static function displaySingleDino(DinoEntity $dinos) : string {
      *
      * @return string - concatenated strings of multiple dinosaur images, names and diets
      */
-    public static function displayMultipleDinos(array $dinos) : string {
+    public static function displayMultipleDinos(array $dinos): string {
         $result = '';
         foreach ($dinos as $dino) {
             $result .= self::displaySingleDino($dino);
         }
         return $result;
     }
+
+
+    /**
+     * Displays a single dino with added detail of height, weight, killer rating, intelligence, and age
+     *
+     * @param DinoEntity $dino
+     *
+     * @return string concatenated string of single dinosaur containing information regarding species, diet, an image, and the above-mentioned statistics
+     */
+    public static function displaySingleDinoDetails(DinoEntity $dino): string {
+
+            $result = '<div class="dinoCard">';
+            $result .= '<h2>' . $dino->getSpecies() . '</h2>';
+            $result .= '<div class="imageContainer">';
+            $result .= '<img src= "' . $dino->getImageUrl() . '" alt="An image of a ' . $dino->getSpecies() . '"/>';
+            $result .= '</div>';
+            $result .= '<div class="dietContainer">';
```

Comment:
I think that before the image, it would be nice to also have the text saying the diet type (carnivore/ herbivore etc.) like we have on the main page, instead of just the image. ie. by adding in the getDiet